api_tests/nodes/views/test_node_files_list.py

```diff
@@ -435,6 +439,276 @@ def test_files_list_contains_relationships_object(self):
         assert res.status_code == 200
         assert 'relationships' in res.json['data'][0]
 
+class TestGVNodeFileList(ApiTestCase):
```

Comment:
This should probably use the [gv_fakes](https://github.com/CenterForOpenScience/osf.io/blob/70e19d43b49f6d0f46d5f688b593bc3f52720de6/osf_tests/external/gravy_valet/gv_fakes.py) if possible so that we can minimize the number of places we need to make changes if anything in the GV API changes

Reply:
fixed